src/test/get_pipeline_metadata_response_test.cpp

```diff
@@ -236,14 +245,25 @@ TEST_F(GetPipelineMetadataResponse, ModelVersionNotLoadedYet) {
     EXPECT_EQ(ovms::GetModelMetadataImpl::buildResponse(pipelineDefinition, &response, manager), ovms::StatusCode::MODEL_VERSION_NOT_LOADED_YET);
 }
 
-TEST_F(GetPipelineMetadataResponse, DISABLED_PipelineNotLoadedAnymore) {
-    // TODO: Implement once waitForLoaded is fully added to PipelineDefinition class
-    EXPECT_TRUE(false) << "Not implemented";
+TEST_F(GetPipelineMetadataResponse, PipelineNotLoadedAnymore) {
+    this->pipelineDefinition.getPipelineDefinitionStatus().handle(ValidationFailedEvent());
+    auto status = ovms::GetModelMetadataImpl::buildResponse(pipelineDefinition, &response, manager);
+    ASSERT_EQ(status, ovms::StatusCode::PIPELINE_DEFINITION_NOT_LOADED_ANYMORE) << status.string();
```

Comment:
Shouldn't it be `PIPELINE_DEFINITION_NOT_LOADED_YET`?

Reply:
The same logic is currently implemented for models. If model fails to load it is in LOADING, ERROR state meaning we return MODEL_NOT_LOADED_YET status.